packages/blockchain-wallet-v4-frontend/src/components/BuySell/Failure.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+import { FormattedMessage } from 'react-intl'
+import { Text, Link } from 'blockchain-info-components'
+import { path } from 'ramda'
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  width: 100%;
+  div:last-of-type {
+    margin-top: 10px;
+  }
+`
+
+const Loading = (e) => (
+  <Container>
+    <Text weight={300} size='16px'>
+      <FormattedMessage id='buysell.general.failure.message' defaultMessage='Sorry, an error has occurred while connecting to your exchange partner.' />
```

Comment:
Shouldn't we add a "retry" link?

Reply:
Not sure how I feel about this. Perhaps for the first step this would be a viable option. If something fails in the middle of the flow, a user's best bet is to start over.